docs/README.md

```diff
@@ -2,6 +2,16 @@
 
 This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
+## Installing `node`
+
+- Checking if node is already present in the system (should print the version number), from the command line:
+
+  ```bash
+   node -v
+  ```
+
+- To install node visit [nodejs.org](https://nodejs.org/en/download/)
+
```

Comment:
node is required for yarn? The install is all yarn based.

Reply:
Yarn is a package manager for node so you definitely need to install node and yarn first before installing all the dependencies

Reply:
Also ignore this readme - you made changes recently so I did not update stuff here 👍🏽